x-pack/plugin/esql/qa/server/single-node/src/yamlRestTest/resources/rest-api-spec/test/40_tsdb.yml

```diff
@@ -109,3 +109,35 @@ filter on counter:
       esql.query:
         body:
           query: 'from test | where k8s.pod.network.tx == 1434577921'
+
+---
+stats on aggregate_metric_double:
+  - do:
+      indices.create:
+        index: test2
+        body:
+          settings:
+            index:
+              mode: time_series
+              routing_path: [ dim ]
+              time_series:
+                start_time: 2021-04-28T00:00:00Z
+                end_time: 2021-04-29T00:00:00Z
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+              dim:
+                type: keyword
+                time_series_dimension: true
+              agg_metric:
+                type: aggregate_metric_double
+                metrics:
+                  - max
+                default_metric: max
+
+  - do:
+      catch: /Cannot use field \[agg_metric\] with unsupported type \[aggregate_metric_double\]/
+      esql.query:
+        body:
+          query: 'FROM test2 | STATS max(agg_metric) BY dim '
```

Comment:
I'd also add a test that's just `FROM test2` and assert that it spits out `null` and `unsupported` doc type. It'd need a document in the index, but I think that's fine.

Reply:
Done.